plugin/advisor/config/sql-review.dev.yaml

```diff
@@ -29,22 +29,22 @@ ruleList:
   - type: naming.index.uk
     level: WARNING
     payload:
-      format: "^uk_{{table}}_{{column_list}}$"
+      format: "^$|^uk_{{table}}_{{column_list}}$"
       maxLength: 64
   - type: naming.index.pk
     level: WARNING
     payload:
-      format: "^pk_{{table}}_{{column_list}}$"
-      maxLength: 64
+      format: "^$|^pk_{{table}}_{{column_list}}$"
+      maxLength: 63 # https://www.postgresql.org/docs/current/limits.html
```

Comment:
According to the link, PostgreSQL is always 63, even for other rules like UK, idx?

Reply:
Right PG always <= 63. For other rules we need to support MySQL, and it doesn't has build-in length limit. Do you think we also need to update other rules to 63 by default?